stroageassignment.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            assignment
+        </title>
+        <meta name="assignment" width="mobile=width inital-scale=1.0">
+        <style>
+            .form{
+                background-color: rgb(138, 209, 159);
+                justify-content: center;
+                text-align: center;
+                width: 20%;
+                height: auto;
+            }
+            body{
+                background-color: rgba(249, 249, 109, 0.783);
+            }
+            .buttons{
+                flex-direction: row;
+                flex-wrap: wrap;
+                display: flex;
+                margin : 2%;
+            }
+            .buttons form{
+                border-radius: 3%;
+                margin: 3%;
+
+            }
+        </style>
+    </head>
+    <body>
+        <div class="buttons">
+        <form class="form">
+            <p>sessionStorage</p>
+            <p id="session">0</p>
+            <button onclick="clickCounter()" type="button">button</button>
+        </form><br>
+        <form class="form">
+            <p>localStorage</p>
+            <p id="local">0</p>
+
+            <button onclick="clickCount()" type="button">button</button>
+        </form>
+        </div>
+
+        <script>
+            function clickCounter(){
+               if(sessionStorage.clickCounter){
```

Comment:
Rather than checking over here, you can create and initialize the value with 0. if it's not present in the storage.